Podcast/RecommendedEpisodesOuterTableViewCell.swift

```diff
@@ -15,6 +15,8 @@ protocol RecommendedEpisodesOuterTableViewCellDataSource {
 
 protocol RecommendedEpisodesOuterTableViewCellDelegate{
     func recommendedEpisodesOuterTableViewCell(cell: RecommendedEpisodesOuterTableViewCell, didSelectItemAt indexPath: IndexPath)
+    func recommendedEpisodesOuterTableViewCellPushTagViewController(tagViewController: TagViewController)
```

Comment:
table view cell should not create an controller, this should be more like: recommendedEpisodesOuterTableViewCellDidTapTag(tag: Tag)